jekyll/_cci2/gh-bb-integration.md

```diff
@@ -186,27 +186,30 @@ CircleCI will use the SSH key you added.
 
 Bitbucket does not currently provide CircleCI with an API
 to create user keys.
-It is still possible to create a user key
+However, it is still possible to create a user key
 by following this workaround:
 
-1. In your project's settings
-on the "Checkout SSH Keys" page,
-open the "Inspect Element" feature
-in your browser's developer tools.
+1. In the CircleCI application,
+go to your project's settings.
 
-2. Click "Create User Key".
```

Comment:
No quotes, just initial caps that match the caps used in the UI and say 'click the Create User Key button'.